install/tools/setupMeshEx.sh

```diff
@@ -86,10 +89,15 @@ function istioDnsmasq() {
 # - name of the k8s cluster.
 function istioClusterEnv() {
    local K8S_CLUSTER=${1:-${K8S_CLUSTER}}
+   local ISTIO_NS=${ISTIO_NAMESPACE:-istio-system}
+   local CP_AUTH_POLICY=${CONTROL_PLANE_AUTH_POLICY:-MUTUAL_TLS"
 
    # TODO: parse it all from $(kubectl config current-context)
    CIDR=$(gcloud container clusters describe ${K8S_CLUSTER} ${GCP_OPTS:-} --format "value(servicesIpv4Cidr)")
    echo "ISTIO_SERVICE_CIDR=$CIDR" > cluster.env
+   echo "ISTIO_SYSTEM_NAMESPACE=$ISTIO_NS" >> cluster.env
```

Comment:
who/what is using those?

Reply:
This is needed to construct discovery address, it is extracted by pilot-agent and used to construct the uri to verify for mTLS